### ISSUE TITLE
GUI list-box% callback doesn't work as advertised on Windows 10

### ISSUE THREAD

**Issue by tramboi** [racket/racket#3030]
Hello reader :)
"(send a-list-box select ...)" invokes the controll callback despite the documentation saying 
"The control’s callback procedure is not invoked."

Cheers,
Bertrand

```#lang racket/gui

(define frame (new frame% [label "Bug?"]))

(define list-box (new list-box%
                      (label "List Box")
                      (parent (new horizontal-panel%
                                   (parent frame)
                                   (style (list 'border))))
                      (choices (list "Item 0"
                                     "Item 1"
                                     "Item 2"))
                      (style (list 'single
                                   'column-headers))
                      (columns (list "First Column"))
                      (callback (lambda (lb e)
                                  (printf "This sould not be printed ('~a event received) ~n" (send e get-event-type))))
                      ))

(send frame show #t)
(send list-box select 0)
```

**Comment by mflatt** [comment]
On Mac OS? If so, this bug has been fixed for the upcoming v7.6 release. If you'd like to give the pre-release a try, it's at https://pre-release.racket-lang.org .

Thanks for the report

**Comment by tramboi** [comment]
Hello,
sorry for the incomplete report, I didn't think for one sec that it could be important.
It's on Windows 10.
Thanks for the answer :)

**Comment by tramboi** [comment]
I can confirm that the 7.5.0.900 in prereleases still has the bug.

**Comment by mflatt** [comment]
I really should have checked Windows when fixing for Mac OS.

I've fixed this problem as racket/gui@ebf4cba60a2328062ab769e636df60075be55de9, but the repair won't make it to a release until v7.7.
